popup.css

```diff
@@ -0,0 +1,489 @@
+.expandcard {
+  display: none;
+  background-color: rgb(193, 199, 208, 0.5);
+  font-family: "Poppins", sans-serif;
+  backdrop-filter: blur(5px);
+  position: fixed;
+  padding-top: 15px;
+  top: 0;
+  right: 0;
+  width: 100vw;
+  height: 100vh;
+}
+
+.container {
+  width: 100%;
+  margin: 114px auto;
+  display: grid;
+  gap: 88px;
+  font-family: "Poppins", sans-serif;
+}
+
+.card-body {
+  border-radius: 16px;
+  background-color: #fff;
+  padding: 16px;
+  width: 86%;
+  margin-top: 10%;
+  margin-bottom: 10%;
+  font-family: "Poppins", sans-serif;
+}
+
+.card-body:hover {
+  animation-name: card-body;
+  animation-duration: 3s;
+}
+
+@keyframes card-body {
+  from {
+    background-color: #e5e9f2;
+  }
+
+  to {
+    background-color: #fff;
+  }
+}
+
+/* Popup Styling start */
+.modal-body {
```

Comment:
- [ ] Kindly note that your design is hiding due to overflow, you might try to rectify this by making the `.modal-body` `overflow` css property to 'scroll', and you can eventually hide the scroll bar if it looks ugly to you 😄, but at least it will be scrollable.  check [this](https://blog.logrocket.com/hide-scrollbar-without-impacting-scrolling-css/) for more information. 😉

- Mobile version
![mobile](https://user-images.githubusercontent.com/41163571/233775018-275a1c86-f40e-4f14-b073-9a793d5261d7.gif)

- Desktop version
  - Normal screen
   ![mobile](https://user-images.githubusercontent.com/41163571/233775138-7607ad45-1b79-4a9d-997b-bfd0a70f2b27.gif)

  - My small sweet screen 🤗
   <img width="845" alt="image" src="https://user-images.githubusercontent.com/41163571/233775327-a2ff526f-3d7e-4f8c-96a0-c5bc6c309e45.png">